TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/ArgumentViewHelper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace TYPO3\Fluid\ViewHelpers;
+
+/*
+ * This file is part of the TYPO3.Fluid package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Fluid\Core\ViewHelper\AbstractViewHelper;
+
+/**
+ * "ARGUMENT" -> only has an effect inside of "RENDER". See Render-ViewHelper for documentation.
```

Comment:
A little usage example would make sense (similar to the ones in other ViewHelpers)